### PR TITLE
adds a navbar

### PR DESCRIPTION
**Pull request by MarissaLZ**
### In This PR

- Adds a `<Navbar/>` component to `<App/>`
- Adds @emotion/react: ^11.9.3 and @emotion/styled: ^11.9.3  since Material UI components would not work without these dependencies.
- Uses the default Material UI styling

`<Navbar/>`

https://user-images.githubusercontent.com/86141440/182040122-be7505c8-a55a-4ee6-a651-64c7172465ed.mov

### CODE REVIEW

**Review by tarikutk — APPROVED** [review]
Looks Good as initial commit. but we still need to make it more response. for example we may need to use menu bar for mobile users because there will no enough space to show all list in row.

**Comment by MarissaLZ** [comment]
Yes I agree that it will need to be more responsive! I'll put that in the team Trello board. This is what the team agreed on so far, but I also think the navigation bar will need to eventually have drop down menus and routing set up.

**Review by MarcusCHill — APPROVED** [review]
The navbar looks good to me! One thing I may add would be comments to your code. Although the react component is short and fairly simple it is always nice to have comments.

File: src/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { AppBar, Toolbar, Stack, Typography, Button } from "@mui/material"
+import React from "react"
+
+export function Navbar() {
+  const navItems = ["Adopt", "Volunteer", "Donate", "About"]
```
Comment (mnca16) [inline src/Navbar.js:5]: I like the array of strings for for the navbar entries, looks clean and functional. Great job!

**Review by mnca16 — APPROVED** [review]
The NavBar looks good to me! It covers everything we discussed our last meeting.

**Comment by IreneS1** [comment]
Everything looks great to me! I agree with the above comments.

**Review by IreneS1 — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
-
+import logo from "./logo.svg"
```
Comment (Jay-zar) [inline src/App.js:1]: I think we can take out this line importing the React placeholder logo, but otherwise looks good!

**Review by Jay-zar — APPROVED** [review]

**Pull request merged**